### PR TITLE
solution

### PR DESCRIPTION
**Pull request by kq333**
[DEMO LINK](https://kq333.github.io/react_decompose/)

### CODE REVIEW

File: src/components/Article/Article.jsx [hunk 1]
```diff
@@ -1,6 +1,70 @@
+/* eslint-disable react/react-in-jsx-scope */
```
Comment (d-wasilewski) [inline src/components/Article/Article.jsx:1]: this shouldn't be necessary

File: src/components/Article/Article.jsx [hunk 2]
```diff
@@ -1,6 +1,70 @@
...
 // import the React library here
 // import a css file containig article styles
 
 // Create an Article function returning the HTML of article block
 
 // Add a default export statement for Article component to use it in the other files
+
+export const Articles = () => (
```
Comment (d-wasilewski) [inline src/components/Article/Article.jsx:9]: you should add default exports

`export default Article` on the bottom of the file

File: src/components/Header/Header.jsx [hunk 3]
```diff
@@ -1,6 +1,20 @@
...
+
+export const Header = () => (
+  <>
+    <header className="header">
+      <h1 className="header__title">Site Name</h1>
+      <nav className="navigation">
+        <a className="navigation__link" href="#about">About</a>
+        <a className="navigation__link" href="#services">Services</a>
+        <a className="navigation__link" href="#contact">Contact</a>
+      </nav>
```
Comment (d-wasilewski) [inline src/components/Header/Header.jsx:17]: You can move this to another folder too

**Review by d-wasilewski — CHANGES_REQUESTED** [review]

File: src/components/Article/Article.jsx (hunk 1 above)
Comment (kq333) [reply src/components/Article/Article.jsx:1]: When i remove this line, my whole component starts  lightings on red.

File: src/components/Article/Article.jsx [hunk 4]
```diff
@@ -1,6 +1,72 @@
+/* eslint-disable react/react-in-jsx-scope */
 // import the React library here
 // import a css file containig article styles
 
 // Create an Article function returning the HTML of article block
 
 // Add a default export statement for Article component to use it in the other files
+
+const Articles = () => (
+  <>
```
Comment (d-wasilewski) [inline src/components/Article/Article.jsx:10]: unnecessary fragment

File: src/components/Article/Article.jsx (hunk 1 above)
Comment (d-wasilewski) [reply src/components/Article/Article.jsx:1]: what does it say?

Comment (d-wasilewski) [reply src/components/Article/Article.jsx:1]: maybe you should import react in every file

File: src/components/Navigation/Navigation.jsx [hunk 5]
```diff
@@ -0,0 +1,13 @@
+/* eslint-disable react/react-in-jsx-scope */
+const Navigation = () => (
```
Comment (d-wasilewski) [inline src/components/Navigation/Navigation.jsx:2]: you should import scss and react here

**Review by d-wasilewski — CHANGES_REQUESTED** [review]

**Review by d-wasilewski — CHANGES_REQUESTED** [review]

**Review by d-wasilewski — APPROVED** [review]

**Pull request closed without merging**